eclipse-collections-code-generator/src/main/resources/impl/list/mutable/synchronizedPrimitiveList.stg

```diff
@@ -409,6 +412,14 @@ public class Synchronized<name>List
             return this.getMutable<name>List().collectWithIndex(function, target);
         }
     }
+<if(primitive.specializedStream)>
+
+    @Override
+    public Spliterator.Of<name> primitiveSpliterator()
```

Comment:
This method should have similar Javadoc warnings here on the method and at the top of the class, just like iterator().